setup.py

```diff
@@ -1,10 +1,9 @@
 import os
-from pathlib import Path
 import sys
 from setuptools import setup, find_packages
 
 # allow setup.py to be run from any path
-os.chdir(str(Path(__file__).absolute().parent))
+os.chdir(os.path.abspath(os.path.dirname(os.path.realpath(__file__))))
```

Comment:
Was there a reason for removing pathlib?  Python2 compatibility?

Reply:
Yes, using `pathlib` with **Python 2.7** raise an `ImportError`.

```
>>> from pathlib import Path
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pathlib
```

According with the [documentation](https://docs.python.org/3/library/pathlib.html) `pathlib` is *New in version 3.4*.